pkg/pillar/types/global.go

```diff
@@ -4,9 +4,10 @@
 package types
 
 import (
-	"strconv"
-
+	"fmt"
 	log "github.com/sirupsen/logrus"
+	"strconv"
```

Comment:
This order of includes is unique. I thought the normal was to have the standard ones first (in alphabetical order), then a blank line, then others (also in alphabetical order)